### ISSUE TITLE
Userscript has incorrect name if `author` in package.json is specified as object

### ISSUE THREAD

**Issue by 7nik** [lisonge/vite-plugin-monkey#38]
The `author` field in `package.json` can be specified as a string or the object
```ts
{
  name: string,
  email?: string,
  url?: string,
}
```

In the latter case, the `author.name` is used as the userscript name instead of the `name` field from `package.json` which is of course incorrect.

## Example

If `package.json` has the following fields:
```js
"name": "script_name",
"author": {
  "name": "author_name"
},
```
the userscript will be compiler with the name `author_name` instead of `script_name`.

**Comment by lisonge** [comment]
<https://github.com/lisonge/vite-plugin-monkey/blob/2db294c01461a7134aa675d5a88bdabad18cd835/packages/vite-plugin-monkey/src/node/_util.ts#L113>
should be
```js
target.author = rawTarget.author?.name;
```

**Comment by lisonge** [comment]
fixed by [v2.10.2](https://github.com/lisonge/vite-plugin-monkey/blob/v2.10.2/packages/vite-plugin-monkey/CHANGELOG.md)
